src/Rendered.sol

```diff
@@ -75,8 +84,12 @@ abstract contract Rendered {
      * @param _data The data to be set for the token.
      */
     function _setTokenData(uint256 _tokenId, bytes calldata _data) internal {
```

Comment:
Can you create a test file called Rendered.t.sol in the test directory and test this function using dapptools framework